mlflow/projects/__init__.py

```diff
@@ -46,10 +46,20 @@
 _PROJECT_TAR_ARCHIVE_NAME = "mlflow-project-docker-build-context"
 _MLFLOW_DOCKER_TRACKING_DIR_PATH = "/mlflow/tmp/mlruns"
 
-
 _logger = logging.getLogger(__name__)
 
 
+def _resolve_experiment_id(experiment_name=None, experiment_id=None):
+    exp_id = experiment_id
```

Comment:
Could you add a short doc-string comment to this method indicating its contract?